src/routes/Ecosystem/Common/VaultField.styled.tsx

```diff
@@ -0,0 +1,61 @@
+import Box from '@mui/material/Box';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
+import { Typography as CustomTypography } from '../../../components/atomic/Typography/Typography';
+
+export const VaultFieldBox = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: ${({ theme }) => theme.spacing(4)};
+`;
+
+export const TitleBox = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0px;
+  gap: ${({ theme }) => theme.spacing(4)};
+`;
+
+export const TitleTypography = styled(Typography)`
+  font-family: 'PixelOperatorMono', monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 130%;
+
+  letter-spacing: 0.02em;
+
+  color: #e5e1f9;
+`;
+
+export const VaultMetricsBox = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0px;
+  gap: ${({ theme }) => theme.spacing(16)};
```

Comment:
This is huge gap. 64px are you sure about this?

Reply:
Yes, it's because some label doesn't consider that gap.

Reply:
What does it mean? Can you explain?